### DISCUSSION TITLE
Apache container just isn't there

### DISCUSSION THREAD

**Discussion by SageSphinx63920** [nextcloud/all-in-one#1844] [Questions] [answered]
My nextcloud can't start anymore because my apache container just left. Idk where it went but its not there anymore and the AIO manager can't update without it so i dont know what i should do to keep my install. Should i just backup and setup everything up again or is there just a command i can run to pull the container with all env variables, etc again?

**Comment by szaimen** [comment]
Hi, can you open the AIO interface again, reload, stop and start your containers and post the mastercontainer logs here? Thanks!

**Comment by SageSphinx63920** [comment]
The Nextcloud containers won't start again. I clicked on restart but the admin gui just showed the reloading circle and never came back again

**Reply by szaimen** [reply]
Whrn you manually reload, does the circle hide and can you afterwards get the masterxontainer logs and post them here?

**Reply by SageSphinx63920** [reply]
after waiting some time it finally reloaded. Here is the log of the master container https://bin.sageee.xyz/sosuvigifo.coffeescript

**Reply by SageSphinx63920** [reply]
The apache container shows as stopped and this is the error when clicking on the stopped button 
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/69404127/215359938-d8d19723-89c5-40b8-86fb-a1a1e696b670.png">
 and there is no apache container on my docker 
<img width="1505" alt="image" src="https://user-images.githubusercontent.com/69404127/215359961-6a867b71-22af-42e7-903e-43c916d79a3f.png">

**Comment by szaimen** [comment]
Thanks for the logs! It looks like you run into this issue: `error creating overlay mount to /var/lib/docker/overlay2/7d863916d4ee996f65c96bacf098275898cc2795a737e085823 (truncated...)`

See https://www.google.com/search?q=error%20creating%20overlay%20mount%20to for possible fixes to your issue.

**Comment by SageSphinx63920** [comment, marked as answer]
I couldn't find anything fixing my problem. I just installed nextcloud native and I'm now just doing everything on my own

**Reply by szaimen** [reply]
Yeah, sorry that I could not help with this but sometimes people run into issues with Docker that I cannot reproduce since they are caused by their infrastrcuture.

**Reply by szaimen** [reply]
See e.g. https://github.com/nextcloud/all-in-one#disrecommended-vps-providers

**Reply by SageSphinx63920** [reply]
I just installed nextcloud natively on my raspberry pi with a cloudflare tunnel. Works fine now just some little problems with nginx rewriting for CalDAV
